documentation/ADMINISTRATA.rst

```diff
@@ -241,34 +239,13 @@ Test the installed compressed distribution::
     $ cd ..
     $ \rm -rf tmp
```

Comment:
`\rm` --> `rm`

Reply:
I stand corrected: @guyer points out that `\` strips aliases, so `\rm` is a perfectly valid shell command.